.github/workflows/release.yaml

```diff
@@ -0,0 +1,77 @@
+name: Release
+on:
+  push:
+    branches:
+      - master
+      - main
```

Comment:
Any need to include a main branch which does not exist?

Reply:
Yeah, it was always planned to move to main, so ive used the same config where I can